[CI] ChunkedFetchPhaseCircuitBreakerTrippingIT testCircuitBreakerTripsWithConcurrentSearches failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #12158 / almalinux-8-aarch64_checkpart1_platform-support-arm](https://gradle-enterprise.elastic.co/s/uylficsq7l7cu)
- [elasticsearch-periodic-platform-support #12158 / almalinux-8-aarch64_checkpart1_platform-support-arm](https://gradle-enterprise.elastic.co/s/ibausxuq3jc2a)
- [elasticsearch-periodic-platform-support #12153 / amazonlinux-2023_platform-support-aws](https://gradle-enterprise.elastic.co/s/3zolsidq7qlhm)
- [elasticsearch-periodic-platform-support #12148 / amazonlinux-2023_platform-support-aws](https://gradle-enterprise.elastic.co/s/dxrh7qingmmaq)
- [elasticsearch-pull-request #140520 / part-1](https://gradle-enterprise.elastic.co/s/c4h56zyo7xw6q)
- [elasticsearch-pull-request #140520 / part-1](https://gradle-enterprise.elastic.co/s/pfeb4qvcw4cxe)
- [elasticsearch-pull-request #138843 / part-1-windows](https://gradle-enterprise.elastic.co/s/l7a2fshi6kyti)
- [elasticsearch-periodic-platform-support #12128 / almalinux-8_platform-support-unix](https://gradle-enterprise.elastic.co/s/5fpeb3g2fz6bc)

**Reproduction Line:**
```
./gradlew ":server:internalClusterTest" --tests "org.elasticsearch.search.fetch.ChunkedFetchPhaseCircuitBreakerTrippingIT.testCircuitBreakerTripsWithConcurrentSearches" -Dtests.seed=16F5B20D995C1F1B -Dtests.locale=ar-KM -Dtests.timezone=Indian/Mayotte -Druntime.java=26
```

**Applicable branches:**
9.4

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.search.fetch.ChunkedFetchPhaseCircuitBreakerTrippingIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testCircuitBreakerTripsWithConcurrentSearches'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Expected at least one circuit breaker exception
Expected: a value greater than <0>
     but: <0> was equal to <0>
```

**Issue Reasons:**
- [9.4] 2 consecutive failures in step almalinux-8-aarch64_checkpart1_platform-support-arm
- [9.4] 8 failures in test testCircuitBreakerTripsWithConcurrentSearches (2.6% fail rate in 303 executions)
- [9.4] 2 failures in step almalinux-8-aarch64_checkpart1_platform-support-arm (28.6% fail rate in 7 executions)
- [9.4] 2 failures in step amazonlinux-2023_platform-support-aws (28.6% fail rate in 7 executions)
- [9.4] 2 failures in step part-1 (2.5% fail rate in 81 executions)
- [9.4] 4 failures in pipeline elasticsearch-periodic-platform-support (57.1% fail rate in 7 executions)
- [9.4] 2 failures in pipeline elasticsearch-pull-request (2.5% fail rate in 80 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -321,9 +321,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.qa.mixed.EsqlClientYamlIT
   method: test {p0=esql/250_high_cardinality_keyword/Test avg length}
   issue: https://github.com/elastic/elasticsearch/issues/147380
-- class: org.elasticsearch.search.fetch.ChunkedFetchPhaseCircuitBreakerTrippingIT
-  method: testCircuitBreakerTripsWithConcurrentSearches
-  issue: https://github.com/elastic/elasticsearch/issues/147391
 - class: org.elasticsearch.index.reindex.ReindexRelocationOnShutdownIT
   method: testReindexTaskRelocatesOnNodeShutdown
   issue: https://github.com/elastic/elasticsearch/issues/147449

--- a/server/src/internalClusterTest/java/org/elasticsearch/search/fetch/ChunkedFetchPhaseCircuitBreakerTrippingIT.java
+++ b/server/src/internalClusterTest/java/org/elasticsearch/search/fetch/ChunkedFetchPhaseCircuitBreakerTrippingIT.java
@@ -170,7 +170,7 @@ public class ChunkedFetchPhaseCircuitBreakerTrippingIT extends ESIntegTestCase {
                 var client = internalCluster().client(coordinatorNode);
                 var resp = client.prepareSearch(INDEX_NAME)
                     .setQuery(matchAllQuery())
-                    .setSize(4)
+                    .setSize(5)
                     .setAllowPartialSearchResults(false)
                     .addSort(SORT_FIELD, SortOrder.ASC)
                     .get();

